src/components/controller/loader.ts

```diff
@@ -1,43 +1,39 @@
-import { IDataSources } from "../view/sources/sources";
+import { IDataSources } from '../view/sources/sources';
 
 export type Callback<T> = (data: T) => void;
 class Loader {
-    baseLink: string;
-    readonly options: { [key: string]: string };
-
-    constructor(baseLink: string, options: { [key: string]: string }) {
+    constructor(private baseLink: string, private options: { [key: string]: string }) {
         this.baseLink = baseLink;
         this.options = options;
     }
 
-    public getResp(
-        { endpoint, options = {} }: {endpoint: string, options?: object},
-        callback: Callback<IDataSources>) {
+    getResp({ endpoint, options = {} }: { endpoint: string; options?: object }, callback: Callback<IDataSources>) {
         this.load('GET', endpoint, callback, options);
     }
 
     errorHandler(res: Response) {
+        const enum StatusCode {
+            errorUnauthorized = 401,
+            errorNotFound = 404,
+        }
         if (!res.ok) {
-            if (res.status === 401 || res.status === 404)
+            if (res.status === StatusCode.errorUnauthorized || res.status === StatusCode.errorNotFound)
                 console.log(`Sorry, but there is ${res.status} error: ${res.statusText}`);
             throw Error(res.statusText);
         }
-
         return res;
     }
 
     makeUrl(options: object, endpoint: string) {
         const urlOptions: object = { ...this.options, ...options };
-        let url: string = `${this.baseLink}${endpoint}?`;
-
-        (Object.keys(urlOptions) as Array <keyof typeof urlOptions>).forEach((key) => {
+        let url = `${this.baseLink}${endpoint}?`;
+        (Object.keys(urlOptions) as Array<keyof typeof urlOptions>).forEach((key) => {
             url += `${key}=${urlOptions[key]}&`;
         });
-
         return url.slice(0, -1);
     }
 
-    load(method: string, endpoint: string, callback: Callback<IDataSources>, options: Object = {}) {
+    load(method: string, endpoint: string, callback: Callback<IDataSources>, options: object = {}) {
```

Comment:
Please return Object. The name of class should start with a capital letter.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object

Reply:
Done.